client/components/forms/form-label/index.tsx

```diff
@@ -4,7 +4,7 @@ import { Children, FunctionComponent, LabelHTMLAttributes } from 'react';
 
 import './style.scss';
 
-interface Props {
+export interface Props {
```

Comment:
Any reason to export this? I could not find a reference to this in this change.

Reply:
Because of this error. Any types used in a reusable component need to be exported. Otherwise the consumer of the component and its typescript instance cannot figure out the type used in said component.
https://github.com/Automattic/wp-calypso/pull/60436#discussion_r791594895

Reply:
With this fix, the error : https://github.com/Automattic/wp-calypso/pull/60436#discussion_r791594895 Should not have appeared. Can you check again?

Reply:
TIL!